### DISCUSSION TITLE
Who wants to be hired? (Apr 2022)

### DISCUSSION THREAD

**Discussion by Nutlope** [vercel/next.js#35839] [Ideas]
# What is this?
This discussion is for Next.js developers looking for work.

# Candidates

Share your information if you are looking for work. Please use this format:

  Location:
  Remote:
  Willing to relocate:
  Technologies:
  Resume/CV:
  Website:
  Email:

Good luck with interviewing!

---

*If you're looking for last month's discussion, see #34948. We've had another [*success story*](https://twitter.com/Nutlope/status/1475929984174895105) of a developer being hired!* 

If you got hired through this job board, please reach out to me (hassan@vercel.com) so I can send you some swag :)

If you're a company looking to post a role, use #35840.

**Comment by abe312** [comment]
Need **help** with **migrating to next.js**?  or **parallel deployment**(where new code, old code and some CMS is running on the same domain)?

Looking for a React developer who would be **productive** from day 1?

Hi. I'm looking for full-time remote roles in a well-funded stable company (I will work in the company's core office hours. 0 issues with timezones. Been working remotely before covid)

Building websites since 2016, working with React since 2018 and have 2 years of production next.js experience

Super comfortable with Next.js, React, Redux (with sagas), tailwind, material ui, styled-components, SEO schemas, AMP, MERN, **scalable deployment** on GCP/AWS/Vercel, incremental migration and parallel deployment (WordPress blogs, new next.js code and old code working on the same domain)

I value web performance (web vitals) and accessibility and could create high conversion dynamic pages

website: https://abhineet.me

email: contact+vercel@abhineet.me

**Comment by G3root** [comment]
Hello everyone 👋,
I'm a frontend developer from Sri Lanka, looking for a job opportunity in frontend development. 

Tech stacks I'm comfortable working with are:

- React / Next JS / React Native 
- Redux / Zustand / Jotai
- Tailwind CSS / Chakra UI / Mantine UI / Material UI / CSS / CSS-in-JS
- Graphql / Trpc / Rest API
- Prisma
- Typescript
- Solidity / Web 3 (beginner)

Some of my projects I worked on:

- [Eisen Matrix](https://github.com/G3root/Eisen-Matrix) - A prioritization app that uses Eisenhower Matrix, built with React Native.
- [Readme Generator](https://github.com/G3root/readme-generator) - README generator with a drag and drop interface.
- [Maxcell.lk](http://maxcell.lk) - An e-commerce site built with Next JS, Tailwind CSS, Prisma and Trpc.js.
- [Next.js dapp starter ts](https://github.com/G3root/nextjs-dapp-starter-ts) - A fullstack monorepo template to develop Ethereum dapps

Location: Sri Lanka
Remote: Yes
Willing to relocate: Maybe
Resume/CV: [Resume](https://drive.google.com/file/d/1FCdSyX5p2iHHJadHOM4CVh7QXnHoUbGz/view?usp=drivesdk) 
Website: [nafeez.xyz](https://www.nafeez.xyz/)
Email: [nafeesnazik21@gmail.com](mailto:nafeesnazik21@gmail.com)

**Comment by kellenwiltshire** [comment]
Hi Everyone,

I am a passionate full-stack developer specializing in all aspects of Web Development looking for the next step in my journey. I have the experience to create beautiful and functional websites combining the latest frameworks and technologies. I have experience in React, Javascript/Typescript, TailwindCSS, Styled Components, as well as traditional HTML5 and CSS3. I am currently look for a new Web Development career opportunity! 

Location: Ontario, Canada
Remote: Yes
Willing to relocate: No
Technologies: NextJS, React, TailwindCSS, Strapi, PostgresSQL, Emotion, Typescript, Squareup/Shopify, Currently learning Vue
Résumé/CV: [Linkedin](https://www.linkedin.com/in/kellenwiltshire/ )
Website: [kellenwiltshire.com](https://kellenwiltshire.com)
Email: webdevelopment@kellenwiltshire.com

**Comment by Xelaflash** [comment]
Hi all, 
I'm Alex 👋, 
A **freelance frontend developer** with **4 years of Xp,** willing to pivot to a **full-time position** in a tech company/agency. 
I've begun my work journey in the **finance** field, but after a few years, I went to more IT-related role within an international bank in the EU. 
In 2018, I attended a Bootcamp where I learned **Ruby and Ruby on rails**. I've since worked as a **freelancer**. 
In the past 2 years, I've decided to specialize in **Frontend web development** with the **JAMSTACK** and **NextJS**. 
I'm now living in **NYC** and I'm looking to find a Frontend position there. 

Location: NYC, USA
Remote: Yes
Willing to relocate: No
Technologies: NextJS, Gatsby, React, JS, TailwindCSS, styled-components, GraphQL, Headless CMS... [see more](https://alexgwebdev.notion.site/ccd78484faf74078a6cba18f37880225?v=55880afa00284854b98d35610210ccda)
Resume/CV: [Linkedin](https://www.linkedin.com/in/alexgwebdev/) / [Resume](https://alexgwebdev.notion.site/Alexandre-Gros-e06bc5d76e604dadaee7559f07500b30)
Website: [alexgwebdev.com](https://www.alexgwebdev.com)
Email: [contact@alexgwebdev.com](mailto:contact@alexgwebdev.com)

**Reply by samuelfuchs** [reply]
This portfolio website looks really cool! 😎

**Comment by akapetano** [comment]
Hi everyone! 

I'm Andrija, a front-end developer, who specializes in Next.js and React. My developer journey started in July 2021, and as a beginner, I have strong technical skills with HTML, CSS and JavaScript, which I strive to improve on a daily basis.

Let me tell you a bit about myself. I've been working in higher education at the University of Zadar for almost five years, where I've gained experience with data gathering and processing. Coincidentally, this is also where I discovered my love for web development which started as a mere content-editing task in the scope of our office's website via a CMS. However, I wanted to do more than that; I wanted for our users, the students, to have a superb user experience, but unfortunately, because of some administrative complications, it was difficult to influence the website beyond the scope of the content. Nonetheless, I decided to learn web development, it's front-end to be precise, and after falling in love with it, I decided to quit my job, and to fully commit to finding a job in web development. You can read more about my journey on my [website](https://www.andrijakapetanovic.com/blog/posts/my-first-post-and-the-journey-so-far) which was build with TypeScript, Next.js and Chakra UI. I'm currently building a cryptocurrency website with Next.js, which I'm hoping to launch by the end of May.

I am eager to learn new technologies and to start a career in web development, and here's what I'm offering in return:
- unique qualities which are sometimes lacking in the IT sphere (multilingualism, intercultural competence, eloquence, meticulousness),
- a new and different skill set and experience,
- a breath of fresh creative air, as I love to draw, work with my hands and create stuff,
- fix services, as I love fixing stuff and finding solutions to difficult problems, be that in the digital or physical sphere.

Location: Zadar, Croatia
Remote: Yes
Willing to relocate: Yes
Technologies: NextJS, React, TypeScript, JavaScript, Chakra UI, Tailwind CSS, styled-components, SCSS, CSS, HTML, Git, Github, Linux, Figma, Node.js (currently learning)
Resume/CV: [Resume](https://www.andrijakapetanovic.com/about/resume)/[PDF](https://drive.google.com/file/d/1KzbCqzzOZM5YQzUJcx9eB7nFzisHf2gz/view?usp=sharing)
Website: [andrijakapetanovic.com](https://www.andrijakapetanovic.com/)
Email: [andrija.kapetanovic@gmail.com](andrija.kapetanovic@gmail.com)
